grc/core/FlowGraph.py

```diff
@@ -371,7 +371,7 @@ def import_data(self, data):
             )
 
             if isinstance(block, blocks.DummyBlock):
-                print('Block id "%s" not found' % block_id)
+                block.add_error_message('Block id "{}" not found.'.format(block_id))
```

Comment:
IMO this error should be attached to fg, `self.add_error_message`

Reply:
I changed this because I wanted to get an error message into grc either in the errors and warnings window or into the Properties window of the selected block, and not into the terminal window running grc.
But neither block.add_error_messages nor self.add_error_message delivers an error message.
I have no idea where this message goes to.

So I'd propose to this message completely.